detail.scss

```diff
@@ -0,0 +1,72 @@
+$background-color:#FFF;
+$border-color:#BED23C;
+$main-font:"Roboto";
+
+html { 
```

Comment:
Don't style HTML tags directly. I think for this you need the `*` operator instead, that will target all the elements 😄

Reply:
Done